src/main/scala/com/cognite/spark/datasource/CursorsCursorIterator.scala

```diff
@@ -0,0 +1,19 @@
+package com.cognite.spark.datasource
+
+import com.softwaremill.sttp.Uri
+import io.circe.generic.auto._
```

Comment:
Seems like this isn't needed here.

Reply:
Thanks, you're good at catching those. :)